Doppler.ImageAnalyzer.Api/Controllers/ImageAnalyzerController.cs

```diff
@@ -19,6 +20,29 @@ public ImageAnalyzerController(IMediator mediator, ILogger<ImageAnalyzerControll
         _imageAnalysisResultService = imageAnalysisResultService;
     }
 
+    [HttpGet("{analysisResultId}")]
+    public async Task<ActionResult<Response<AnalysisResultResponse>>> GetAnalysisResult([FromRoute] string analysisResultId)
+    {
+        List<ImageAnalysisResponse>? analysisResult;
+        try
+        {
+            analysisResult = await _imageAnalysisResultService.GetAsync(analysisResultId);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Unexpected failure obtaining analysis result.");
+            return HandleResponse(mapImageAnalysisResponseList(null, HttpStatusCode.InternalServerError, null, analysisResultId), "Returned image analysis result");
```

Comment:
I suggest naming the parameter where is passed the `null` value, is easier to code review.

This suggestion applies to all parameters where the value passed is a constant or is not easy to infer what the parameter is.

Also the test `"Returned image analysis result"` could be moved to a variable or constant to be reused

Reply:
Done: https://github.com/FromDoppler/doppler-image-analyzer/pull/324/commits/4134040bb47e013c303883c11f48573d8133284d